main.py

```diff
@@ -56,7 +57,7 @@ def __init__(self, token: str, base_channel_id: int, base_voice_id: int, kikisen
 
         # mesm_json_syntax_conceal = 0sages.json (時報json) の読み込みを試みる
         # msg_dictのkeyはstr型です、int型で呼び出そうとしないで()
-        with codecs.open("messages.json", 'r', 'utf-8') as json_file:
+        with codecs.open(os.getcwd() + "./messages.json", 'r', 'utf-8') as json_file:
```

Comment:
```suggestion
        with codecs.open(os.getcwd() + "/messages.json", 'r', 'utf-8') as json_file:
```
ディレクトリ名が変なことになってァ！するゾ